### ISSUE TITLE
Improved textures recompressed from the PS2 version

### ISSUE THREAD

**Issue by the-gaybanana-corporation** [elishacloud/Silent-Hill-2-Enhancements#1066]
(sorry if this doesn't comply with the usual formatting for posting issues on this page, pretty new to posting on github in general)

This isn't inherently an issue with the actual mod itself but should bring this project even more up to par with the PS2 version (I know you guys had your final update but hear me out). In short, I've been playing around with the Silent Hill 2 Texture Explorer and realized that there were some large quality gaps between the textures found in the PC version and the PS2 version, especially with the PC's DXT3-compressed transparent textures.
So for a few months now, I've been working on porting these textures over, re-compressing any transparent textures with DXT5, as well as removing the white backgrounds found in non-transparent textures from the PC version which mess with the DXT compression algorithm.
I've attached some screenshots of the old and new textures and a little sample mod that replaces the textures found in the starting restroom for anyone in or outside of the enhanced team to compare and contrast with the old sh2e textures. These should work from my testing.
![texture example 8 original](https://github.com/user-attachments/assets/805387c1-4545-4da1-8ff7-29a69fb9e092)
![texture example 8 new](https://github.com/user-attachments/assets/120af4fb-5295-4de3-98da-43d99dfcc444)
[sh2e.zip](https://github.com/user-attachments/files/17141816/sh2e.zip)

**Comment by TommvinHuwaltzky** [comment]
How about AI-upscaling them while you're at it...? :-333

**Comment by the-gaybanana-corporation** [comment]
> How about AI-upscaling them while you're at it...? :-333

lmao i probably could but thats outside the scope of what im tryin to accomplish plus i heard theres a bunch of difficulties when it comes to replacing this game's textures, hell the game outright crashes when i replace a texture with a different data size value no matter if its smaller or larger (those values are hardcoded), so normal size it is!

**Comment by TommvinHuwaltzky** [comment]
Well, they managed to replace the ground-texture when James is digging for the key, soooo..... :-3

**Comment by certainmorning** [comment]
Numerous requests were made to add option to inject HD textures, but it seems EE Team simly don't care about it. Really sad.

**Comment by the-gaybanana-corporation** [comment]
> Numerous requests were made to add option to inject HD textures, but it seems EE Team simply doesn't care about it. Really sad.

for me personally I dont mind having the ingame texture size unchanged that much though i do very much appreciate the upscaled ui elements

**Comment by Polymega** [comment]
> Numerous requests were made to add option to inject HD textures, but it seems EE Team simply doesn't care about it. Really sad.

It is too much of an undertaking. If it could be my fulltime job and I got paid to do it, I'd do it in a heartbeat. As it stands, AI upscalers do not meet my level of standards for environment textures, and a large part of that issue is due to how the imagery within any given texture file is arranged:

![image](https://github.com/user-attachments/assets/f8605352-2f1b-428b-a481-9c8a0d6a0d28)

^ AI upscalers simply can't understand what's going on with all this nonsense to make something fruitful. To do this right, the textures would (nearly) all need to be remade by a human hand.

**Comment by ProtonBlast** [comment]
True. AI upscaled textures don't look great and would bloat the game's file size.

**Comment by robertesaum** [comment]
> How about AI-upscaling them while you're at it...? :-333

I'm recreating by hand all the textures that I and other people on VGTP found the source images for, but we need help finding more sources and matches.

As a placeholder, I'm using some upscales done by AI (I used 3 models and filtered the best results), plus various touch-ups I did in photoshop (mainly fixing melted text, transparency problems, seam problems, etc). Some other textures I've recreated from scratch.

If you'd like to see some of the results, I'm posting a bit of this process on YouTube. I don't know if it's allowed to send links here, if not I'll delete it without any problems:
 
https://www.youtube.com/@robertesaum3/community (unfortunately YouTube compresses the quality of the images a lot)
https://www.youtube.com/watch?v=_g_WzFdWvFo (this video is quite old, I've already managed to recreate several other textures that weren't shown there, as well as a significant improvement in the quality of the placeholders).

![1](https://github.com/user-attachments/assets/ae5b3f39-49e4-4736-81b3-ffae70060b8a)
![2](https://github.com/user-attachments/assets/bee57a09-d38b-44a9-971d-6063ad1cab97)

**Comment by the-gaybanana-corporation** [comment]
lmao i did not intend this thread to go into a whole thing about ai upscaling

**Comment by the-gaybanana-corporation** [comment]
> > Numerous requests were made to add option to inject HD textures, but it seems EE Team simply doesn't care about it. Really sad.
> 
> It is too much of an undertaking. If it could be my fulltime job and I got paid to do it, I'd do it in a heartbeat. As it stands, AI upscalers do not meet my level of standards for environment textures, and a large part of that issue is due to how the imagery within any given texture file is arranged:
> 
> ![image](https://private-user-images.githubusercontent.com/27395260/374385559-f8605352-2f1b-428b-a481-9c8a0d6a0d28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEzMDQwMTAsIm5iZiI6MTczMTMwMzcxMCwicGF0aCI6Ii8yNzM5NTI2MC8zNzQzODU1NTktZjg2MDUzNTItMmYxYi00MjhiLWE0ODEtOWM4YTBkNmEwZDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTExVDA1NDE1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2ODYzOGQwMzI5NDdmZmE1N2U4OTIzNmZlZWFjOWUwNTMzMGIwMWZkMWY5MzE5OGQ1MTg1NDIwOTAxNWY3OTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5H362Jyx5nB-35cOFeWHggsOau-87IMl9LKY3b8e98c)
> 
> ^ AI upscalers simply can't understand what's going on with all this nonsense to make something fruitful. To do this right, the textures would (nearly) all need to be remade by a human hand.

however i should say i found a really useful way to use ai and its to clean up all the blockiness that comes with editing dxt-compressed textures.
During my work in fixing up the enhanced edition textures myself, i realized that the edited textures for the boss room in the prison kinda highlighted the compression heavily and it looked a lil ugly in the transition from the black areas to green areas.
So to fix this, i ran it thru an ai upscaling program, downscaled the ai image to the original texture's resolution, layered the original texture over the ai-upscaled one in gimp and used the eraser tool on the original to clean out all the blockiness, i think it did a decent job without ruining the original texture's art direction. Heres an example

![ps189_1](https://github.com/user-attachments/assets/6f6a7440-b3a8-44ac-a9d0-83326d62fb6b)
my new fixed version

![ps189_1](https://github.com/user-attachments/assets/eda95fb9-6da4-46d7-85d6-3cb3ffdb5929)
enhanced edition version

**Comment by TommvinHuwaltzky** [comment]
> True. AI upscaled textures don't look great and would bloat the game's file size.

Look at stuff like Moguri for FFIX; AI-upscaled textures can look fantastic when done right.
And SH2 is old and pretty slim, even with all the EE-enhancements. Nowadays file-size shouldn't be a problem.
People have to cope with CoD's 100GB-updates on the regular, so this is nothing in comparison.

**Comment by the-gaybanana-corporation** [comment]
>  this should definitely go on the PCGamingWiki (:

Lmao thanks man appreciate the kind words, idk how possible that would be but im def gonna try pushing this on the pcgamingwiki once i get around to testing this mod even further.

**Comment by TommvinHuwaltzky** [comment]
> idk how possible that would be

Well, either add it yourself or put a request on the "Talk"-page of the SH2-article. Done.
